deploy.sh

```
@@ -0,0 +1,17 @@
+
+EC2_HOST="ubuntu@ec2-3-36-57-197.ap-northeast-2.compute.amazonaws.com"
+PEM_KEY_PATH="/Users/spqje/key-jenson.pem"
+TARGET_DIR="~/home/ubuntu/app"
+JAR_NAME="spring-roomescape-payment-0.0.1-SNAPSHOT.jar"
+TARGET_JAR_PATH="$TARGET_DIR/$JAR_NAME"
+
+./gradlew clean build -x test
+
+scp -i "$PEM_KEY_PATH" "build/libs/$JAR_NAME" "$EC2_HOST:$TARGET_JAR_PATH"
+
+ssh -i "$PEM_KEY_PATH" "$EC2_HOST" << EOF
```

Shellcheck (0.10.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)